### PR TITLE
SES 6 backport of Tuning Guide

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/tuning-architecture.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
```
Comment (asettle) [inline xml/tuning-architecture.xml:13]:
```suggestion
   <dm:release>SES 6</dm:release>

File: xml/tuning-ceph-tiered-storage.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
  <title>Cache Tiering</title>
  <info>
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-   <dm:editurl>https://github.com/SUSE/doc-ses/edit/maintenance/ses6/xml/</dm:editurl>
+   <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
```
Comment (asettle) [inline xml/tuning-ceph-tiered-storage.xml:13]:
```suggestion
   <dm:editurl>https://github.com/SUSE/doc-ses/edit/maintenance/ses6/xml/</dm:editurl>

File: xml/tuning-ceph-tiered-storage.xml [hunk 3]
```diff
@@ -268,10 +268,10 @@
    </step>
    <step>
     <para>
-     Turn the machine into a &ceph; node using &deepsea;. Install the software
+     Turn the machine into a &ceph; node using &cephsalt;. Install the software
```
Comment (asettle) [inline xml/tuning-ceph-tiered-storage.xml:271]:
```suggestion
     Turn the machine into a &ceph; node using &deepsea;. Install the software

File: xml/tuning-ceph-tiered-storage.xml [hunk 4]
```diff
@@ -268,10 +268,10 @@
...
      and configure the host machine as described in
-     <xref linkend="salt-adding-nodes"/>. Let us assume that its name is
-     <replaceable>node-4</replaceable>. This node needs to have 4 OSD disks.
+     <xref linkend="adding-node"/>. Let us assume that its name
```
Comment (asettle) [inline xml/tuning-ceph-tiered-storage.xml:273]:
```suggestion
     <xref linkend="salt-adding-nodes"/>. Let us assume that its name is

File: xml/tuning-ceph-tiered-storage.xml [hunk 5]
```diff
@@ -268,10 +268,10 @@
...
+     is <replaceable>node-4</replaceable>. This node needs to have 4 OSD disks.
```
Comment (asettle) [inline xml/tuning-ceph-tiered-storage.xml:274]:
```suggestion
     <replaceable>node-4</replaceable>. This node needs to have 4 OSD disks.

File: xml/tuning-ceph-tuning.xml [hunk 6]
```diff
@@ -10,7 +10,7 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
```
Comment (asettle) [inline xml/tuning-ceph-tuning.xml:13]:
```suggestion
   <dm:release>SES 6</dm:release>

File: xml/tuning-intro.xml [hunk 7]
```diff
@@ -10,7 +10,7 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
```
Comment (asettle) [inline xml/tuning-intro.xml:13]:
```suggestion
   <dm:release>SES 6</dm:release>

File: xml/tuning-intro.xml [hunk 8]
```diff
@@ -207,7 +207,7 @@
    <itemizedlist>
      <listitem>
        <para>
-         &productname; &productnumber;
+         &productname; &prevproductnumber;
```
Comment (asettle) [inline xml/tuning-intro.xml:210]:
```suggestion
         &productname; &productnumber;

File: xml/tuning-os-tuning.xml [hunk 9]
```diff
@@ -10,7 +10,7 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
```
Comment (asettle) [inline xml/tuning-os-tuning.xml:13]:
```suggestion
   <dm:release>SES 6</dm:release>

File: xml/tuning-architecture.xml [hunk 10]
```diff
@@ -327,6 +324,13 @@
...
+       <note>
+         <para>
+           Keep in mind that we recommend reserving 4&nbsp;GB for the WAL device. The recommended
+           size for DB is a total of 64&nbsp;GB for most workloads. See <xref linkend="rec-waldb-size"/>
+           for more information.
+         </para>
+       </note>
        <para>
          Making the decision to provision fast space for the fourth tier of
          RocksDB is something completely related to the expected metadata
```
Comment (lproven) [inline xml/tuning-architecture.xml:336]:
```suggestion
         RocksDB is entirely related to the expected metadata
```

Comment (lproven) [reply xml/tuning-architecture.xml:336]: Also in the rest of the para...

> somewhere in a mild

somewhere *from* mild to moderate

> whether the fourth tier makes sense to move to faster media.

whether it makes sense to move the fourth tier to faster media.

File: xml/tuning-ceph-tuning.xml [hunk 11]
```diff
@@ -409,18 +438,17 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
...
        utilizing very low-latency media, such as NVMe, for the cephfs metadata pool.
      </para>
      <para>
-       Utilizing the <command>ceph daemon perf dump</command> command (for example,
-       <command>ceph daemon mds.node3 perf dump</command>), there is a significant
+       Utlizing the <command>ceph-daemon perf dump</command> command, there is a significant
        amount of data that can be examined for the &ceph; Metadata Servers.
-       It should be noted that the MDS performance counters only apply to metadata
-       operations. The actual I/O path is from clients straight to OSDs.
+       It should be noted that the MDS perf counters only apply to metadata
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:450]: Why the change from the full word to an abbreviation? I would favour the full word.

File: xml/tuning-ceph-tuning.xml [hunk 12]
```diff
@@ -409,18 +438,17 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
...
+       operations. The actual IO path is from clients straight to OSDs.
      </para>
      <para>
        CephFS supports multiple metadata servers. These servers can operate
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:454]:
```suggestion
       &cephfs; supports multiple metadata servers. These servers can operate
```

File: xml/tuning-ceph-tuning.xml [hunk 13]
```diff
@@ -676,21 +704,21 @@ radosgw-admin bucket reshard --bucket={bucket name} --num-shards={prime number}
...
        operations for the index. This can be adjusted with the
        <literal>rgw_bucket_index_max_aio</literal> parameter.
      </para>
      </sect2>
      <sect2>
        <title>Window Size</title>
      <para>
-       When working with larger objects, increasing the size of the &rgw;
+       When working with larger objects, increasing the size of the &ogw;
        windows for <literal>put</literal> and get can help with performance.
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:716]:
```suggestion
       windows for <literal>put</literal> and <literal>get</literal> can help with performance.
```

File: xml/tuning-ceph-tuning.xml [hunk 14]
```diff
@@ -266,28 +265,58 @@ min_alloc_size_hdd=65536
...
+       Increasing the &bluestore; cache size can improve performance with
+       many workloads. While &filestore; OSDs cache data in the kernel's page
+       cache, &bluestore; OSDs cache data within the memory allocated by the
+       OSD daemon itself. The OSD daemon will allocate memory up to its
+       memory target (as controlled by the <literal>osd_memory_target</literal> parameter),
+       and this determines the potential size of the &bluestore; cache.
+       The &bluestore; cache is a read cache that by default is populated when
+       objects are read. By setting the cache's minimum size higher than the default,
+       it is guaranteed that the value specified will be the minimum cache
+       available for each OSD. The idea being that more low-probability cache
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:280]:
```suggestion
       available for each OSD. The idea is that more low-probability cache
```

File: xml/tuning-ceph-tuning.xml [hunk 15]
```diff
@@ -409,18 +438,17 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
      a client, there are some client options that are covered in the
      client section.
    </para>
-   <sect2 xml:id="mds-tuning">
+   <sect2>
      <title>MDS Tuning</title>
      <para>
        In filesystems with millions of files, there is some advantage to
        utilizing very low-latency media, such as NVMe, for the cephfs metadata pool.
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:445]:
```suggestion
       utilizing very low-latency media, such as NVMe, for the &cephfs; metadata pool.
```

File: xml/tuning-ceph-tuning.xml (hunk 13 above)
Comment (lproven) [reply xml/tuning-ceph-tuning.xml:716]: Also note 2 typos in a later line to which I cannot comment directly:

> allowing up to two devices or nods to fail while still proecting

allowing up to two devices or **nodes** to fail while still **protecting**

File: xml/tuning-os-tuning.xml [hunk 16]
```diff
@@ -283,10 +263,7 @@ salt '*' state.apply my_kerntune
 salt '*' cmd.run 'ip link set bond0 mtu 9000'
 </screen>
           <para>
-            To set this persistently, use YaST to set the MTU for the bonded interface
-            or adjust the <filename>/etc/sysconfig/network/ifcfg-*</filename> file's MTU settings
-            and restart the network. Ensure that jumbo frames are enabled on all
-            switches, routers, and network cards that will transport these packets.
+            To set this persistently, utilize YaST to set the MTU for the bonded interface.
```
Comment (lproven) [inline xml/tuning-os-tuning.xml:266]:
```suggestion
            To set this persistently, utilize &yast; to set the MTU for the bonded interface.
```

Comment (lproven) [reply xml/tuning-os-tuning.xml:266]: Additional changes in older text:

>  We recommend to test at least random 4&nbsp;kB, random 64&nbsp;kB, and sequential

 We recommend **testing** at least random 4&nbsp;kB, random 64&nbsp;kB, and sequential

>  any caches that may apply. We recommend to use the <literal>ramp-time</literal>

 any caches that may apply. We recommend **using** the <literal>ramp-time</literal>

>  and needs to only be done in conjunction with the

 and needs to be done **only** in conjunction with the

File: xml/tuning-ceph-tuning.xml (hunk 11 above)
Comment (asettle) [reply xml/tuning-ceph-tuning.xml:450]: They're called perf counters: https://docs.ceph.com/docs/master/dev/perf_counters/
